core/modes/command_and_dictation_mode.talon

```diff
@@ -1,4 +1,5 @@
-not mode: sleep
+mode: command
+mode: dictation
```

Comment:
Can you please update the breaking changes file to note that if users have their own custom modes defined, they will no longer be able to use the standard words for toggling back to command or dictation mode unless they add the additional modes to the context header of this file (or add the corresponding command(s) to their custom mode directly).